fdbserver/include/fdbserver/TestTLogServer.actor.h

```diff
@@ -106,15 +108,16 @@ struct TLogTestContext : NonCopyable, public ReferenceCounted<TLogTestContext> {
 
 	// paramaters
 	std::string diskQueueBasename;
-	int numCommits;
-	int numTagsPerServer;
-	int numLogServers;
-	int initVersion;
-	int recover;
+	uint32_t numCommits;
+	uint32_t numTags;
+	uint32_t numLogServers;
+	uint32_t initVersion;
+	uint32_t recover;
 
 	// tLog state
 	std::vector<Reference<TLogContext>> pTLogContextList;
 	TestTLogOptions tLogOptions;
+	std::map<std::tuple<uint32_t, uint32_t>, std::vector<Version>> commitHistory;
```

Comment:
better add a comment explaining this

Reply:
I'll have a follow up PR and will add this.